packages/@aws-cdk/aws-ec2/test/integ.vpc-lookup.ts

```diff
@@ -1,48 +1,58 @@
+import { StringParameter } from '@aws-cdk/aws-ssm';
 import * as cdk from '@aws-cdk/core';
-import { IntegTest } from '@aws-cdk/integ-tests';
+import { ExpectedResult, IntegTest } from '@aws-cdk/integ-tests';
 import * as ec2 from '../lib';
+import { SubnetType } from '../lib';
 
-const appWithVpc = new cdk.App();
-const stack = new cdk.Stack(appWithVpc, 'StackWithVpc', {
-  env: {
-    region: 'eu-west-1',
-    account: '123456',
-  },
-});
+const app = new cdk.App();
+const account = process.env.CDK_INTEG_ACCOUNT ?? process.env.CDK_DEFAULT_ACCOUNT;
 
-const testVpc = new ec2.Vpc(stack, 'MyVpc', {
-  vpcName: 'my-vpc-name',
-});
-
-const appUnderTest = new cdk.App();
-const stackLookup = new cdk.Stack(appUnderTest, 'StackUnderTest', {
+const stackLookup = new cdk.Stack(app, 'StackUnderTest', {
   env: {
-    region: 'us-east-2',
-    account: '123456',
+    region: 'us-east-1',
+    account: account,
   },
+  crossRegionReferences: true,
 });
 
-const vpcFromVpcAttributes = ec2.Vpc.fromVpcAttributes(stackLookup, 'VpcFromVpcAttributes', {
+const vpcFromLookup = ec2.Vpc.fromLookup(stackLookup, 'VpcFromLookup', {
```

Comment:
This new test is correct, I think. Can you add a comment explaining why there's no actual `Vpc` being provisioned here? In the old version of this test there's an actual `Vpc`, but it's unused by the test. This new version removes the misleading `Vpc`, which is good, but it's not obvious that the reason why there's no Vpc is that we're using dummy props for a lookup, since the lookup won't succeed in a single deployment anyway